frontend/src/components/TransactionsList.tsx

```diff
@@ -28,79 +52,182 @@ function TransactionsList() {
 
-                      {/* {!!tx.totalAmountFiat && (
+                        {/* {!!tx.totalAmountFiat && (
                         <p className="text-xs text-gray-400 dark:text-neutral-600">
                           ~{tx.totalAmountFiat}
                         </p>
                       )} */}
+                      </div>
                     </div>
                   </div>
-                </div>
-              </div>
+                </CredenzaTrigger>
+                <CredenzaContent className="slashed-zero">
+                  <CredenzaHeader>
+                    <CredenzaTitle>
+                      {type == "outgoing" ? "Sent Bitcoin" : "Received Bitcoin"}
+                    </CredenzaTitle>
+                  </CredenzaHeader>
+                  <CredenzaBody>
+                    <div className="flex items-center mt-6">
+                      {type == "outgoing" ? (
+                        <div
+                          className={
+                            "flex justify-center items-center bg-orange-100 dark:bg-orange-950 rounded-full w-10 h-10 md:w-14 md:h-14"
+                          }
+                        >
+                          <ArrowUpIcon
+                            strokeWidth={3}
+                            className="w-6 h-6 md:w-8 md:h-8 text-orange-400 dark:text-amber-600 stroke-orange-400 dark:stroke-amber-600"
+                          />
+                        </div>
+                      ) : (
+                        <div className="flex justify-center items-center bg-green-100 dark:bg-emerald-950 rounded-full w-10 h-10 md:w-14 md:h-14">
+                          <ArrowDownIcon
+                            strokeWidth={3}
+                            className="w-6 h-6 md:w-8 md:h-8 text-green-500 dark:text-emerald-500 stroke-green-400 dark:stroke-emerald-500"
+                          />
+                        </div>
+                      )}
+                      <div className="ml-4">
+                        <p className="text-xl md:text-2xl font-semibold">
+                          {Math.floor(tx.amount / 1000)}{" "}
+                          {Math.floor(tx.amount / 1000) == 1 ? "sat" : "sats"}
+                        </p>
+                        {/* <p className="text-sm md:text-base text-gray-500">
+                          Fiat Amount
+                        </p> */}
+                      </div>
+                    </div>
+                    <div className="mt-8">
+                      <p className="dark:text-white">Date & Time</p>
+                      <p className="text-muted-foreground">
+                        {dayjs(tx.settled_at).toString()}
+                      </p>
+                    </div>
+                    <div className="mt-6">
+                      <p className="dark:text-white">Fee</p>
+                      <p className="text-muted-foreground">
+                        {tx.fees_paid} {tx.fees_paid == 1 ? "sat" : "sats"}
```

Comment:
In LND fees_paid is equivalent to tx.amount, have to check the backend before merging this.

Reply:
we need a better way to do the sat/sats. Maybe we should start considering translations

Reply:
I was thinking of that too, let's post on slack